### PR TITLE
0.6.0

### PR DESCRIPTION
**Pull request by thomashli**
increment cartography version

### CODE REVIEW

**Review by achantavy — APPROVED** [review]

**Comment by elfqi** [comment]
Addressing the Cursor review:

**Done**
- Finished LB label migration in `c8t_ec2_exposure.json` and `test_c8t_lateral_propagation.py`: `:LoadBalancer` / `:LoadBalancerV2` → `:AWSLoadBalancer` / `:AWSLoadBalancerV2`.
- Adopted upstream Python **3.13.14** in the production `Dockerfile` (aligned with `dev.Dockerfile` / Cartography 0.139.1) rather than pinning dev back to 3.12.

**Process**
- Will merge only when CI is green.
- After deploy: smoke-test Sentinel GitHub/Azure on staging (still on the test-plan checklist).

**Out of scope for this sync PR**
- Prism `AttackPathELBv2MaxHops` 10→15 — tracked separately; not blocking this PR.

**Reminder:** please merge as a **merge commit**, not squash, so the upstream merge-base stays correct for the next Cartography sync.

**Pull request merged**
